pkgs/by-name/hd/hdrop/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
```

Comment:
This stule doesn't seem to match the style used in most packages.

Reply:
I formatted it now with [nixpkgs-fmt](https://nix-community.github.io/nixpkgs-fmt/)